### PR TITLE
Update contributing guide with a new merging strategy

### PR DESCRIPTION
**Pull request by DarkEld3r**

### CODE REVIEW

File: src/contributing.md [hunk 1]
```diff
@@ -62,14 +62,9 @@ a pencil icon at the top of the page.
...
-or code moves with actual code changes.
-
-Commit messages should be verbose by default consisting of a short subject line
-(50 chars max), a blank line and detailed explanatory text as separate paragraph(s);
-unless the title alone is self-explanatory (like “Correct typo in init.rs”)
-then a single title line is sufficient.
-Commit messages should be helpful to people reading your code in the future,
-so explain the reasoning for your decisions.
+or code moves with actual code changes. However, we merge pull requests using the
+"squash and merge" strategy, so commit messages are only important for the review
```
Comment (slowli) [inline src/contributing.md]: Nitpick: using fancy quotes (“squash and merge”) could be better.

Comment (Maria-Nosyk) [reply src/contributing.md]: I believe in this case a "squash-and-merge" punctuation will be correct

File: src/contributing.md [hunk 2]
```diff
@@ -62,14 +62,9 @@ a pencil icon at the top of the page.
 
 In general [commits should be atomic](https://en.wikipedia.org/wiki/Atomic_commit#Atomic_commit_convention)
 and diffs should be easy to read. For this reason do not mix any formatting fixes
-or code moves with actual code changes.
-
-Commit messages should be verbose by default consisting of a short subject line
-(50 chars max), a blank line and detailed explanatory text as separate paragraph(s);
-unless the title alone is self-explanatory (like “Correct typo in init.rs”)
-then a single title line is sufficient.
-Commit messages should be helpful to people reading your code in the future,
```
Comment (dmitry-timofeev) [inline src/contributing.md]: So folks no longer have to write descriptive commit messages? :trollface:

Comment (DarkEld3r) [reply src/contributing.md]: Yeap. :wink:

**Pull request merged**